libvmaf/src/feature/third_party/funque/x86/hbd_resizer_avx2.c

```diff
@@ -346,6 +345,7 @@ void hbd_vresize_avx2(const int **src, unsigned short *dst, const short *beta, i
         __m256i accum_23_0 = _mm256_add_epi64(mul2_0, mul3_0);
         __m256i accum_01_4 = _mm256_add_epi64(mul0_4, mul1_4);
         __m256i accum_23_4 = _mm256_add_epi64(mul2_4, mul3_4);
+
```

Comment:
can we clean up these unrelated whitespace changes from this pull request and leave only the actually modified files?